### PR TITLE
[INLONG-8791][TubeMQ] Add log level and log path configuration API to Tubemq-client-go

### LINKED ISSUE #8791: [Improve][TubeMQ] tubemq-client-go lacks log level configuration API
### Description

现在 github.com/apache/inlong/inlong-tubemq/tubemq-client-twins/tubemq-client-go 的日志级别是固定的warn，系统可以设置日志级别来输出日志，排查问题。

======================================================================

Now the log level of github.com/apache/inlong/inlong-tubemq/tubemq-client-twins/tubemq-client-go is fixed to warn, which is inconvenient to check.

The system can be improved by increasing the log level configuration API, which can output different detailed logs according to needs, which is convenient for troubleshooting.

### Use case

_No response_

### Are you willing to submit PR?

- [x] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by XiaoYou201**
### Prepare a Pull Request

- Title Example: [INLONG-8791][TubeMQ] Add log level and log path configuration API to Tubemq-client-go

- Fixes #8791 

### Motivation

The PR #8793 has a little problem, I solve it.

### Modifications

- Add validation to SetLogPath and SetLogLeve.

- Add log level and log path configuration API.

### CODE REVIEW

**Review by TszKitLo40 — DISMISSED** [review]

File: inlong-tubemq/tubemq-client-twins/tubemq-client-go/config/config.go [hunk 1]
```diff
@@ -143,6 +144,10 @@ type Config struct {
 		// AfterFail is the heartbeat timeout after a heartbeat failure.
 		AfterFail time.Duration
 	}
+	Log struct {
+		LogPath  string
+		LogLevel string
```
Comment (lvanneo) [inline inlong-tubemq/tubemq-client-twins/tubemq-client-go/config/config.go:149]: Suggest adding comments.

File: inlong-tubemq/tubemq-client-twins/tubemq-client-go/log/config.go [hunk 2]
```diff
@@ -39,3 +44,40 @@ var defaultConfig = &OutputConfig{
...
+// SetLogPath set log path
+func SetLogPath(path string) error {
+	err := isValidLogPath(path)
+	if err != nil {
+		return err
+	}
+	defaultConfig.LogPath = path
+	return nil
+}
+func isValidLogPath(path string) error {
```
[the remark this answers is not in the document]
Comment (XiaoYou201) [reply inlong-tubemq/tubemq-client-twins/tubemq-client-go/log/config.go]: Thanks suggestion, fixed it.

File: inlong-tubemq/tubemq-client-twins/tubemq-client-go/config/config.go [hunk 3]
```diff
@@ -620,3 +638,17 @@ func WithConsumePosition(consumePosition int) Option {
...
+		c.Log.LogLevel = level
+	}
+}
+
+// WithLogPath set log path
+func WithLogPath(path string) Option {
+	return func(c *Config) {
+		c.Log.LogPath = path
+	}
+}
```
[the remark this answers is not in the document]
Comment (XiaoYou201) [reply inlong-tubemq/tubemq-client-twins/tubemq-client-go/config/config.go]: Thanks suggestion, I add some codes to make this configuration take effect.

File: inlong-tubemq/tubemq-client-twins/tubemq-client-go/config/config.go (hunk 1 above)
Comment (XiaoYou201) [reply inlong-tubemq/tubemq-client-twins/tubemq-client-go/config/config.go]: Thanks suggestion, fixed it.

**Review by lvanneo — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**